### ISSUE TITLE
Investigate the usefulness of and maybe Update the Video Tutorial

### ISSUE THREAD

**Issue by mykeels** [changeweb/Unifiedtransform#315]
**Is your feature request related to a problem? Please describe.**
The [video tutorial](https://vimeo.com/334331502) for setting up this project may be outdated, and need to be updated.

**Describe the solution you'd like**
- Investigate how useful the video tutorial is
- If outdated, update the video for the latest version of this software

**Describe alternatives you've considered**
Not doing it. But that's not very helpful

**Additional context**
I'd like to do this myself within the week.

**Comment by changeweb** [comment]
It would be great!

**Comment by mykeels** [comment]
As a prep, I've written this [in-depth guide for setting up the project, manually](https://mykeels.medium.com/unifiedtransform-getting-started-in-2021-286842edacad) on Windows 10.

https://mykeels.medium.com/unifiedtransform-getting-started-in-2021-286842edacad

**Comment by changeweb** [comment]
This is very detailed and good. Go ahead!

BTW, I think each features of the application should be independent of each other and modular. We need to use Service Containers and to design new database schema. We need to change one feature at a time. What do you think?

**Comment by mykeels** [comment]
It's been two weeks, and I've made no progress on the video. If anyone is up for it, please go ahead using [the guide](https://mykeels.medium.com/unifiedtransform-getting-started-in-2021-286842edacad).

@changeweb you're talking about refactoring the application to be modular? Like each feature lives in a folder, with its own service, controller, utils, etc, instead of having a single controllers folder that contains all controllers?

**Comment by changeweb** [comment]
Yes. But is it a good choice? I am planning to build a Beta version in a new Branch using Laravel 8.x, Vue.js, and Bootstrap 5. I would like to use more standard approach this time (Like using more Service providers, Interfaces, Resources, Traits). Besides, I am also planning to make a Simple Docker image in addition to docker-compose file with only two services: unifiedtransform-app, mysql. What do you think?

**Comment by adrianmartinezverax** [comment]
Thank you, docker is working on my linux machine, but having some errors when I run php artisan migrate. here are the errors below.

   Illuminate\Database\QueryException 

  SQLSTATE[HY000] [1045] Access denied for user 'superadmin'@'localhost' (using password: YES) (SQL: select * from information_schema.tables where table_schema = unified_transform_db and table_name = migrations and table_type = 'BASE TABLE')

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:671
    667|         // If an exception occurs when attempting to run a query, we'll format the error
    668|         // message to include the bindings with SQL, which will make this exception a
    669|         // lot more helpful to the developer instead of just the database's errors.
    670|         catch (Exception $e) {
  > 671|             throw new QueryException(
    672|                 $query, $this->prepareBindings($bindings), $e
    673|             );
    674|         }
    675| 

      +35 vendor frames 
  36  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

**Comment by theblackhand123** [comment]
The new version with Bootstrap 5 will be super awesome! I can't wait to test it!! Good work!!

**Comment by changeweb** [comment]
@theblackhand123 I have been working with a new version of Unifiedtransform using Bootstrap 5 and Laravel 8.x. UI and workflow are redesigned. I am trying to follow SOLID principles this time. Hope I can quickly release it so that many would contribute.

**Comment by JuanVqz** [comment]
Wow!
This is great good news, Right now I'm not involved in PHP/Laravel world but it will be a pleasure to share my thoughts about the Unified Transform v2, I can take it a look, through. (since you have a **super** simple docker setup) Congrats @changeweb :+1:

**Comment by theblackhand123** [comment]
Great news! I look forward to a more sleek and an even better UI and a seamless workflow. Cheers!

**Comment by changeweb** [comment]
To @mykeels @JuanVqz @theblackhand123 @gerpollo2000 @norbybaru @anishojha @handhikadj @rowild @luciano-jr @paulomcnally @Atiladanvi @P-Shakibafar @fergthh and all the people who gave their precious time and effort in this project, I am glad to inform you that a completely new version of the project has been released under the version series v2.X.

You may checkout the newly released v2.0 and I appreciate all feedbacks. Thank you.

**Comment by gerpollo2000** [comment]
Thanks Hasib

These are awesome news, I still have a previous demo running on my server,
I'll try to update it with the new version this week, I hope everything
goes ok.
https://edumexico.com.mx/Unifiedtransform/public/

Thanks for your effort and dedication to you and all other collaborators
for the great work you had done.
Kudos

Gerry

**Comment by handhikadj** [comment]
@changeweb well... with all due respect, I think your good news is not relevant with this issue.
It might be better if we talk about that on Discussion tab, probably.
Or even better, boot up a Discord server, on announcement channel for example.
Still good news tho

**Comment by changeweb** [comment]
> @changeweb well... with all due respect, I think your good news is not relevant with this issue. It might be better if we talk about that on Discussion tab, probably. Or even better, boot up a Discord server, on announcement channel for example. Still good news tho

Thanks. You are right. I have created a Discord server. I will add it in the README.md. Here is the server link: https://discord.gg/8sz6kpup99

**Comment by theblackhand123** [comment]
Testing right up! Awesome UI and great work once again! @changeweb

**Comment by Atiladanvi** [comment]
Hy @changeweb congratulations. In the next year i will get some time to create a PR with internationalization stubs.
